pkgs/tools/security/ghidra/extensions/findcrypt/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGhidraExtension,
+}:
+let
+  version = "3.0.1";
+in
+buildGhidraExtension {
+  pname = "findcrypt";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "antoniovazquezblanco";
+    repo = "GhidraFindcrypt";
+    rev = "v${version}";
```

Comment:
Iirc I was told that the Rev should be a string literal. I think this is related to the auto update bot that generates PRs when a new version gets released.

Reply:
I don't think I've ever seen this recommendation before. I know rev is usually set as a string literal when you're working with unstable versions, but I'd love to know more about this, and maybe even add it to https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#sources. Do you happen to remember where you got it from?

Reply:
```
$ grep -Rni rev | grep -Rni '${version}' | wc -l
25517
```
I'm pretty sure this pattern is pretty common...

Reply:
Turns out I remembered it [the wrong way around](https://github.com/NixOS/nixpkgs/pull/315672#discussion_r1699211138) 
sorry for the useless comment, it was a drive-by review from my phone and I couldn't check.